cmd/mysterium_client/command_run/command.go

```diff
@@ -68,6 +69,8 @@ func (cmd *CommandRun) Run(options CommandOptions) (err error) {
 		return err
 	}
 
+	go client_local_api.Bootstrap(options.LocalApiAddress)
```

Comment:
for me `Bootstrap` relates to a configuration routine that runs once before actual business logic could run.
so I'd rather have `client_local_api.Boostrap()` and `go client_local_api.Run(options.LocalApiAddress)` here.

Reply:
Yeah could be some DI approach:
- factory factories/bootstraps `TequilapiServer` to be sure that everything is OK
- here just `TequilapiServer.Start()`

Reply:
I am kinda missing control if of subprocess status:
- `Start() error`
- `Wait() error`
- main programs combines Waiting of 2 subroutines. see `cmd/mysterium_fake/mysterium_fake.go`.